flutter/test/default_integrations_test.dart

```diff
@@ -257,6 +257,40 @@ void main() {
       expect(options.release, 'foo.bar@1.2.3+789');
       expect(options.dist, '789');
     });
+
+    test('sets app name as in release if packagename is empty', () async {
+      final options = SentryFlutterOptions(dsn: fakeDsn);
```

Comment:
we can improve the testing of this class by using the `Fixture` pattern, see file_system_transport_test.dart or load_contexts_integrations_test.dart

a `sut` that returns a new instance of `LoadReleaseIntegration` and takes a `PackageInfo` as param, so we reuse the method creation of `LoadReleaseIntegration` and `SentryFlutterOptions`